pkgs/tools/networking/ssl-proxy/go120-compatibility.patch

```diff
@@ -0,0 +1,18 @@
+Check whether User-Agent is defined before trying to override it
+
+Since Go 1.20 [1], ReverseProxy no longer adds a User-Agent header to forwarded
+requests.
+
+[1] https://github.com/golang/go/commit/f001df540b3fc66a475985c1b7c810e7df063c8f
+
+--- a/reverseproxy/reverseproxy.go
++++ b/reverseproxy/reverseproxy.go
+@@ -32,7 +32,7 @@ func newDirector(target *url.URL, extraDirector func(*http.Request)) func(*http.
+ 		} else {
+ 			req.URL.RawQuery = targetQuery + "&" + req.URL.RawQuery
+ 		}
+-		if _, ok := req.Header["User-Agent"]; !ok {
```

Comment:
TIL about patches. I am going to put up a PR upstream, then :-)

Reply:
@marsam No luck in ~48 hours, and I also sent an email. We can wait longer, or perhaps we can merge right now, and I can put up a PR removing the patch if it gets fixed upstream.

Reply:
Yes.  Thanks a lot for pushing it to upstream.